lib/src/localizations.dart

```diff
@@ -156,6 +156,13 @@ class InsporationLocalizations {
     locale: localeName
   );
 
+  String get errorSignInNoHost => Intl.message(
+      'Unable to resolve host, are you sure this pod supports the API?',
+      name: 'errorSignInNoHost',
+      desc: 'Error message if no host can not be resolved',
+      locale: localeName
+  );
```

Comment:
When you add a new string, please run `script/generate_template_locale` and commit the resulting changes.

Reply:
Also when an existing translation is edited :)